ga4gh/datamodel/variants.py

```diff
@@ -451,9 +451,14 @@ def __init__(self, variantSetId, vcfPath, use_bcf=True):
         self._chromFileMap = {}
         self._metadata = None
         self._callSetIds = None
+        fileList = []
```

Comment:
We could just use counter here: numVariantFiles = 0 initially, and test this at the end.